tests/network/vmi_multus.go

```diff
@@ -1240,7 +1240,7 @@ var _ = SIGDescribe("[Serial]Macvtap", func() {
 				migration := tests.NewRandomMigration(serverVMI.Name, serverVMI.GetNamespace())
 				_ = tests.RunMigrationAndExpectCompletion(virtClient, migration, tests.MigrationWaitTime)
 
-				Expect(libnet.PingFromVMConsole(clientVMI, serverIP)).To(Succeed(), "connectivity is expected *after* migrating the VMI")
+				Expect(libnet.PingFromVMConsole(clientVMI, serverIP, "-c 5", "-w 15")).To(Succeed(), "connectivity is expected *after* migrating the VMI")
```

Comment:
If you think the default should be 15 instead of 10, why you change it only for this check?
It is also worrying that it is not smooth. @maiqueb, can you think of a reason for this?

Reply:
Unfortunately I can't - we'd have to try and figure out where this time is being spent.

Reply:
> If you think the default should be 15 instead of 10, why you change it only for this check?

I did that initially, but then I changed my mind. But maybe it's indeed better to use the same deadline instead of making exceptions.

Reply:
I'm not sure; it depends on whether or not we'll dig more or not. 

Pointing the finger at this test indicating it is weirder than all of them might not be a bad thing after all.

I don't think I have the bandwidth to look into this right now.

Reply:
> Pointing the finger at this test indicating it is weirder than all of them might not be a bad thing after all.

@maiqueb I'm not sure I understand what you mean.

@EdDev The tests started to flake on [2021-06-15](https://prow.ci.kubevirt.io/job-history/gs/kubevirt-prow/pr-logs/directory/pull-kubevirt-e2e-k8s-1.20-sig-network?buildId=1405063916932829184) and it seems to flake at around 50% [now](https://prow.ci.kubevirt.io/job-history/gs/kubevirt-prow/pr-logs/directory/pull-kubevirt-e2e-k8s-1.20-sig-network?buildId=).

I don't see any PR or commit in the tree that would seem to suggest to cause this. I've tried running this test (without the extended dedline) on my local machine and it hasn't failed in 70 attempts (then my machine went to sleep...). 

At this point, we need to make a decision to either quarantine and investigate further or increase the deadline, as this is blocking PRs from merging.